pegasus/helpers/section_api_helpers.rb

```diff
@@ -108,6 +108,7 @@ def self.fields()
       :users__username___username,
       :users__email___email,
       :users__hashed_email___hashed_email,
+      :users__user_type___user_type,
```

Comment:
Reviewing the code, it doesn't appear so.  I believe the only ramification is selecting one more thing in a couple of the queries.